src/renderer/component/link/view.jsx

```diff
@@ -44,6 +46,20 @@ const Link = props => {
     );
   }
 
+  if (span === true) {
+    return (
+      <span
+        className={combinedClassName}
+        href={href || "javascript:;"}
+        title={title}
+        onClick={onClick}
+        {...("style" in props ? { style: style } : {})}
+      >
+        {content}
+      </span>
+    );
+  }
+
   return (
     <a
```

Comment:
this whole thing should probably be structured like this

`let props = { className: combinedClassName, href: href || "javascript:;" etc}`

and then the if statement shold be something like
`span ? <span {...props}> {content} </span> : <a {...props}> {content} </a>`

as per https://zhenyong.github.io/react/docs/jsx-spread.html

let me know if you have any questions!